src/inventory_ui.cpp

```diff
@@ -401,14 +454,329 @@ void inventory_selector::print_right_column() const
     }
 }
 
+std::string *inventory_selector::split_name(const std::string name, std::string* rtn) const {
+    rtn[0] = name;
+    rtn[1] = "";
+    if(name.length() > 24) {
+        for(int i = 24; i > 0; i--) {
+            if(name[i] == ' ') {
+                rtn[0] = name.substr(0, i);
+                rtn[1] = name.substr(i + 1);
+                break;
+            }
+        }
+    }
+    return rtn;
```

Comment:
What's the point of unconditionally returning the parameter? The caller already has that value. Returning `void` should be fine.